src/Illuminate/Database/Eloquent/Collection.php

```diff
@@ -176,7 +176,7 @@ public function intersect($items)
      *
      * @param  string|callable|null  $key
      * @param  bool  $strict
-     * @return static
+     * @return static|\Illuminate\Support\Collection
```

Comment:
Unsure about this one, don't know whether it should be `static|\Illuminate\Support\Collection` or `\Illuminate\Support\Collection`, seeing as this class obviously extends `\Illuminate\Support\Collection`